### PR TITLE
20.04 ubuntu support for devops/dev dockerfiles. Add aws codebuild.

### PR DESCRIPTION
**Pull request by devinleighsmith**
Signed-off-by: Smith <Devin.Smith@quartech.com>

### CODE REVIEW

File: aws_local_codebuild.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+OSNAME=focal
```
Comment (devinleighsmith) [inline aws_local_codebuild.env:1]: sensible defaults in env file.

File: codebuild_build.sh [hunk 2]
```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
```
Comment (devinleighsmith) [inline codebuild_build.sh:1]: aws codebuild local script.

File: devops/Makefile [hunk 3]
```diff
@@ -124,6 +105,11 @@ pre_package: python_version | $(BUILD_DIR)
 $(BUILD_DIR):
 	mkdir -p $@
 
+test_local_aws: image_ci
+	cd .. \
```
Comment (devinleighsmith) [inline devops/Makefile:109]: mmm, will clean this up

File: devops/denv/Dockerfile [hunk 4]
```diff
@@ -1,89 +1,87 @@
...
 ARG indy_plenum_ver=1.13.0~dev1033
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
-RUN echo "deb https://repo.sovrin.org/deb xenial rc" >> /etc/apt/sources.list \
-    && apt-get update && apt-get install -y \
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88 \
+    && echo "deb https://repo.sovrin.org/sdk/deb xenial master" >> /etc/apt/sources.list \
+    && echo "deb https://repo.sovrin.org/deb xenial stable master" >> /etc/apt/sources.list \
+    && echo "deb http://archive.ubuntu.com/ubuntu/ xenial universe main" >> /etc/apt/sources.list \
+    && apt-get update && apt-get install -y --allow-downgrades \
+    python3-jsonpickle=0.9.6 \
```
Comment (devinleighsmith) [inline devops/denv/Dockerfile:20]: had to pin a lot of dependencies required by python3.5 version of plenum to work with 20.04

**Comment by devinleighsmith** [comment]
decided to reuse the images created for the devops related functionality for the docker sync stack to make devops run more similarly to development.

**Comment by WadeBarnes** [comment]
@devinleighsmith, Just so the approach is documented.  You're upgrading to Ubuntu 20.0x first, and then you'll upgrade python.  Is that correct?  Also I'm assuming it's potentially easier at the moment to flesh this out in using the current CI/CD pipeline rather than dealing with multiple things at once and also migrating from Jenkins/AWS code build to GitHub actions.

**Comment by devinleighsmith** [comment]
closing this in favour of: 
https://github.com/sovrin-foundation/token-plugin/pull/409

**Pull request closed without merging**